Task/MCU/Floor1_USART_inputdata/Core/Src/main.c

```diff
@@ -186,6 +185,11 @@ int main(void)
 
     /* USER CODE BEGIN 3 */
 #ifdef R_T_C
+	  if(flag!=0)
```

Comment:
Change variable name to respective of RTC

Reply:
done